src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -450,27 +450,28 @@
-                                sender-network-values)
-                       :always (assoc-in [:wallet :ui :send :receiver-network-values]
-                                receiver-network-values)
-                       :always (assoc-in [:wallet :ui :send :suggested-routes-call-timestamp] now)
-                       :always (update-in [:wallet :ui :send] dissoc :network-links)
-                       token   (assoc-in [:wallet :ui :send :token] token))
-      :json-rpc/call [{:method     "wallet_getSuggestedRoutes"
-                       :params     request-params
-                       :on-success (fn [suggested-routes]
-                                     (rf/dispatch [:wallet/suggested-routes-success suggested-routes
-                                                   now]))
-                       :on-error   (fn [error]
-                                     (rf/dispatch [:wallet/suggested-routes-error error])
-                                     (log/error "failed to get suggested routes"
-                                                {:event  :wallet/get-suggested-routes
-                                                 :error  error
-                                                 :params request-params}))}]})))
+     (when (and to-address from-address amount-in token-id)
+       {:db            (cond-> db
+                               :always (assoc-in [:wallet :ui :send :amount] amount)
+                               :always (assoc-in [:wallet :ui :send :loading-suggested-routes?] true)
+                               :always (assoc-in [:wallet :ui :send :sender-network-values]
+                                                 sender-network-values)
+                               :always (assoc-in [:wallet :ui :send :receiver-network-values]
+                                                 receiver-network-values)
+                               :always (assoc-in [:wallet :ui :send :suggested-routes-call-timestamp] now)
+                               :always (update-in [:wallet :ui :send] dissoc :network-links)
+                               token   (assoc-in [:wallet :ui :send :token] token))
+        :json-rpc/call [{:method     "wallet_getSuggestedRoutes"
+                         :params     request-params
+                         :on-success (fn [suggested-routes]
+                                       (rf/dispatch [:wallet/suggested-routes-success suggested-routes
+                                                     now]))
+                         :on-error   (fn [error]
+                                       (rf/dispatch [:wallet/suggested-routes-error error])
+                                       (log/error "failed to get suggested routes"
+                                                  {:event  :wallet/get-suggested-routes
+                                                   :error  error
+                                                   :params request-params}))}]}))))
```

Comment:
@OmarBasem just a simple feedback, it would help to just flag the change you made here as it is likely small and would be easier to see since the git diff is so large.
Thanks for fixing this issue! 🙏

Reply:
> Hey @BalogunofAfrica, I have been working on fixing the above notification error, and I realized you are updating the getSuggestedRoutes method in your [PR](https://github.com/status-im/status-mobile/pull/20631/files#diff-089fb8d72eefa455e3dc6437b0bc11c3ca3cc0fb4063e20f129b09929aeae3a9L365-R361), so should I close this one?

@J-Son89 The difference is just the `when` condition but I will close this PR anyways at @BalogunofAfrica is already fixing the same issue in his PR